var/spack/repos/builtin/packages/autoconf/package.py

```diff
@@ -18,17 +18,30 @@ class Autoconf(AutotoolsPackage, GNUMirrorPackage):
             preferred=True)
     version('2.62', sha256='83aa747e6443def0ebd1882509c53f5a2133f502ddefa21b3de141c433914bdd')
     version('2.59', sha256='9cd05c73c5fcb1f5ccae53dd6cac36bb8cb9c7b3e97ffae5a7c05c72594c88d8')
-    version('2.13', sha256='f0611136bee505811e9ca11ca7ac188ef5323a8e2ef19cffd3edb3cf08fd791e')
 
     # https://savannah.gnu.org/support/?110396
     patch('https://git.savannah.gnu.org/cgit/autoconf.git/patch/?id=05972f49ee632cd98057a3caf82ebfb9574846da',
           sha256='eaa3f69d927a853313a0b06e2117c51adab6377a2278549b05abc5df93643e16',
           when='@2.70')
+    # tests/foreign.at (Libtool): Be tolerant of 'quote' replacing the older `quote'
+    patch('http://mirrors.mit.edu/gentoo-portage/sys-devel/autoconf/files/autoconf-2.69-fix-libtool-test.patch',
+          sha256='7793209b33013dc0f81208718c68440c5aae80e7a1c4b8d336e382525af791a7',
+          when='@2.69')
+    # Fix autoscan when using Perl 5.17+ (reported already in January 2013)
+    patch('http://mirrors.mit.edu/gentoo-portage/sys-devel/autoconf/files/autoconf-2.69-perl-5.26.patch',
+          sha256='35c449281546376449766f92d49fc121ca50e330e60fefcfc9be2af3253082c2',
+          when='@2.62:2.69')
+    # Fix for newer perl not having "." in @INC:
+    patch('http://mirrors.mit.edu/gentoo-portage/sys-devel/autoconf/files/autoconf-2.69-perl-5.26-2.patch',
+          sha256='a49dd5bac3b62daa0ff688ab4d508d71dbd2f4f8d7e2a02321926346161bf3ee',
+          when='@2.62:2.69')
 
     # Note: m4 is not a pure build-time dependency of autoconf. m4 is
     # needed when autoconf runs, not only when autoconf is built.
     depends_on('m4@1.4.6:', type=('build', 'run'))
     depends_on('perl', type=('build', 'run'))
+    # Because it detects that we patched a file, autoconf-2.62 wants help2man:
+    depends_on('help2man', when='@2.62', type='build')
```

Comment:
Would it be reasonable to assume this would apply to all versions `2.62` and later? In that case `when='@2.62:'` (note the colon at the end).

Reply:
Only 2.62 tries to update the man page in this case (BTW, from 2008).

Newer versions introduced ./build-aux/help-extract.pl, the entire logic changed.
For them, to run help2man, one would have to touch e.g. the `autoscan.w` wrapper, which this patch doesn't do.